src/components/AllServicesDropdown/PlatformServicesLinks.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ChromeLink from '../ChromeLink';
+
+const PlatformServiceslinks = () => {
+  return (
+    <>
+      <ChromeLink href="/ansible/ansible-dashboard" className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
+        Red Hat Ansible Platform
+      </ChromeLink>
+      <ChromeLink href="/insights/dashboard#SIDs=&tags=" className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
```

Comment:
I don't think we want to apply any filter, or even use the empty ones in the URL. 

The URL should be `/insights`. Otherwise, it will do a full page refresh. The app is not registered for such a pathname.